src/Semver.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * This file is part of composer/semver.
+ *
+ * (c) Composer <https://github.com/composer>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace Composer\Semver;
+
+use Composer\Semver\Constraint\VersionConstraint;
+
+class Semver
+{
+    /** @var VersionParser */
+    private static $versionParser;
+
+    /**
+     * @see Composer\Semver\Comparator::greaterThan
+     *
+     * @param string $version1
+     * @param string $version2
+     *
+     * @return bool
+     */
+    public static function gt($version1, $version2)
+    {
+        return Comparator::greaterThan($version1, $version2);
+    }
+
+    /**
+     * @see Composer\Semver\Comparator::greaterThanOrEqualTo
+     *
+     * @param string $version1
+     * @param string $version2
+     *
+     * @return bool
+     */
+    public static function gte($version1, $version2)
+    {
+        return Comparator::greaterThanOrEqualTo($version1, $version2);
+    }
+
+    /**
+     * @see Composer\Semver\Comparator::lessThan
+     *
+     * @param string $version1
+     * @param string $version2
+     *
+     * @return bool
+     */
+    public static function lt($version1, $version2)
+    {
+        return Comparator::lessThan($version1, $version2);
+    }
+
+    /**
+     * @see Composer\Semver\Comparator::lessThanOrEqualTo
+     *
+     * @param string $version1
+     * @param string $version2
+     *
+     * @return bool
+     */
+    public static function lte($version1, $version2)
+    {
+        return Comparator::lessThanOrEqualTo($version1, $version2);
+    }
+
+    /**
+     * @see Composer\Semver\Comparator::equalTo
+     *
+     * @param string $version1
+     * @param string $version2
+     *
+     * @return bool
+     */
+    public static function eq($version1, $version2)
+    {
+        return Comparator::equalTo($version1, $version2);
+    }
+
+    /**
+     * @see Composer\Semver\Comparator::notEqualTo
+     *
+     * @param string $version1
+     * @param string $version2
+     *
+     * @return bool
+     */
+    public static function neq($version1, $version2)
+    {
+        return Comparator::notEqualTo($version1, $version2);
+    }
```

Comment:
Yeah hence I added the ? in the task list to this item. Not sure if we want to offer it or not. It's not harmful though, and it would allow for shorter to write complicated constructs. I'd mostly consider it just a nice-to-have, but I will remove it if majority is against it :-)